dev/bin/setup-hydra-client.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
would it make to look into terraform so we have some parity with prod, instead of having a script? or that would not make sense for dev

Reply:
IMO too much overhead to bring in another tool for dev.